R/pkg/R/DataFrame.R

```diff
@@ -851,6 +849,8 @@ setMethod("nrow",
             count(x)
           })
 
+#' ncol
```

Comment:
Yes, this doesn't seem to be consistent. I reverted these since the description is sufficient to explain.